tests/test_listings.py

```diff
@@ -0,0 +1,95 @@
+"""
+Unit tests for listing methods to have expected columns
+with CashCtrlClient.
+"""
+
+import pandas as pd
+from cashctrl_api import CashCtrlClient
+from test_constants import TAX_COLUMNS, ACCOUNT_COLUMNS, JOURNAL_ENTRIES
+
+# Test function with assertions to check the tax rates DataFrame
+def test_list_tax_rates_to_have_expected_columns():
+    """
+    Test that the DataFrame returned by CashCtrlClient contains all expected columns
+    with the correct data types as specified in TAX_COLUMNS.
+    """
+    # Given: CashCtrlClient returns a DataFrame of tax rates
+    cc_client = CashCtrlClient()
+    tax_rates = cc_client.list_tax_rates()
+
```

Comment:
As a first test, I like to ensure that `taxt_rates` is a pd.DataFrame:
```
asset isinstance(tax_rates, pd.DataFrame), "`tax_rates` is not a DataFrame."
```
(Applies to all tests)

Reply:
fixed